docs/modules/user-guide/partials/proc_limiting-resources-usage.adoc

```diff
@@ -8,9 +8,9 @@
 
 . Specify container memory limit for components
 +
-To specify a container(s) memory limit for `container`, `plugin`, use the `memoryLimit` parameter:
+To specify a container(s) memory limit for `container`, use the `memoryLimit` parameter:
 +
-.Specify container memory limit for components
+.Specify container memory request for components
```

Comment:
I think point 1 and 2 can be combined to say 

To specify a container(s) memory limit for container, use the memoryLimit parameter and to specify a container(s) memory request, use the memoryRequest parameter

95% of the info is the same in the 2 points, even the yaml snippet is the same.

Update the yaml snippet to:

```
 components:
   - name: maven
     container:
      image: eclipe/maven-jdk8:latest
      memoryLimit: 512M
      memoryRequest: 256M
```